backend/game/cards/keep_cards/dice_roll_manipulation_cards/test_acid_attack.py

```diff
@@ -0,0 +1,10 @@
+from game.cards.keep_cards.dice_roll_manipulation_cards.acid_attack import AcidAttack
+
+
+def test_acid_attack_player_add_card(player):
+    AcidAttack().immediate_effect(player, None)
+    assert player.card_on_hand.get("Acid Attack") == "Add [attack] to your roll"
```

Comment:
You're going to write this test for every single keep card in the entire deck?
OR you can put the "add this card to the player" into KeepCard and test it once.